<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckInController</class>
 <widget class="QMainWindow" name="CheckInController">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Check In Operator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">#centralwidget{

background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgb(1, 18, 71), stop:0.5 rgb(0, 58, 120), stop:1 rgb(1, 18, 71));
}

#centralwidget QLabel{
color: white;
}
</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frmTop">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>473</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="lbUsername">
         <property name="text">
          <string>timo</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnLogout">
         <property name="text">
          <string>Logout</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QFrame" name="frmLeft">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">#frmLeft{
border: 3px solid white;
border-radius: 10px;
background: rgb(2, 61, 116);
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Reservation Number</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLineEdit" name="resNum"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>ID Card Number</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLineEdit" name="idCardNum"/>
         </item>
         <item row="4" column="0">
          <widget class="QPushButton" name="btnSearch">
           <property name="text">
            <string>Search</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frmRight">
        <property name="styleSheet">
         <string notr="true">#frmRight{
border: 3px solid white;
border-radius: 10px;
background: rgb(2, 61, 116);
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>280</y>
           <width>91</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Weight</string>
         </property>
        </widget>
        <widget class="Line" name="line">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>260</y>
           <width>591</width>
           <height>16</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QListWidget" name="listWeight">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>350</y>
           <width>171</width>
           <height>121</height>
          </rect>
         </property>
        </widget>
        <widget class="QWidget" name="horizontalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>300</y>
           <width>171</width>
           <height>41</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QDoubleSpinBox" name="sbAddLuggage">
            <property name="value">
             <double>0.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnAddLuggage">
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="verticalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>410</y>
           <width>91</width>
           <height>61</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPushButton" name="btnRemove">
            <property name="text">
             <string>Remove</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnRemoveAll">
            <property name="text">
             <string>Remove All</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="verticalLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>420</x>
           <y>290</y>
           <width>160</width>
           <height>119</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Total Weight</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lbChargeWeight">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>0.00 Kg</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Total Charged</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lbChargeMoney">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Rp. 0.00</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="verticalLayoutWidget_3">
         <property name="geometry">
          <rect>
           <x>240</x>
           <y>60</y>
           <width>91</width>
           <height>191</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Passenger Name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Flight Number</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Flight Description</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Boarding Time</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Gate</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Seat Number</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="verticalLayoutWidget_4">
         <property name="geometry">
          <rect>
           <x>340</x>
           <y>60</y>
           <width>241</width>
           <height>191</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="lbNama">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>&lt;Passenger Name&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lbFlightNumber">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>&lt;Flight Number&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lbSchedule">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>&lt;Flight Description&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lbBoardingTime">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>&lt;Boarding Time&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lbGate">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>&lt;Gate&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lbSeat">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>&lt;Seat Number&gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QPushButton" name="pushButton_2">
         <property name="geometry">
          <rect>
           <x>420</x>
           <y>442</y>
           <width>158</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Check In</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_13">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>20</y>
           <width>111</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Reservation by </string>
         </property>
        </widget>
        <widget class="QLabel" name="lbApplicant">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>20</y>
           <width>451</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>&lt;Applicant&gt;</string>
         </property>
        </widget>
        <widget class="Line" name="line_2">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>60</y>
           <width>20</width>
           <height>201</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>60</y>
           <width>191</width>
           <height>201</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QListWidget" name="listPassenger">
            <property name="font">
             <font>
              <family>MS Shell Dlg 2</family>
              <pointsize>9</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnCheck">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>Check</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbAddLuggage">
            <property name="text">
             <string>Add Extra Luggage</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
